.github/ktlint-message.md

```diff
@@ -0,0 +1,6 @@
+This project follows the [official Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html). We use [ktlint](https://github.com/pinterest/ktlint) to check that, and unfortunately, there is some code that violates the code style. Please go to this PR's *Files* tab and have a look at the inline annotations in the code. To see a list of all lint issues, click *Details* near the failing "❌ Lint / Kotlin" check below.
```

Comment:
```suggestion
This project follows the [official Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html). We use [ktlint](https://github.com/pinterest/ktlint) to check that, and unfortunately, there is some code that mismatches the code style. Please go to this PR's *Files* tab and have a look at the inline annotations in the code. To see a list of all lint issues, click *Details* near the failing "❌ Lint / Kotlin" check below.
```

Reply:
I think that less aggressive word would be better.

Maybe mention also

> just formatting the code inside Android Studio now fixes most lint issues

?

Reply:
Thanks, I applied the nicer wording.

Your other suggestion is already accounted for in the next sentence:

> You can fix most issues automatically by [reformatting the code in Android Studio](https://stackoverflow.com/q/40926920).